pymc/model.py

```diff
@@ -1849,6 +1849,13 @@ def point_logps(self, point=None, round_vals=2):
 def set_data(new_data, model=None, *, coords=None):
     """Sets the value of one or more data container variables.
 
+    When doing prediction or posterior predictive sampling, make sure that
```

Comment:
whats the difference between prediction and posterior predictive sampling?

Reply:
Nothing.. wanted to catch the eye of someone skimming with either word.  Maybe better phrasing here?

Reply:
I would just drop the first prediction mention.

"make sure" is also a bit strong, as it's not required, it's more of a convenience. And maybe also say at the end that the benefit of this is that you only have to `set_data()` `x`, but not both.

Reply:
I would also add an example of resizing `y` alone. MutableData is not useful only in GLMs where this concern for having to resize both variables arises.

Would be strange for me as user seeing the only example cover this.

Reply:
@twiecki that makes sense, I'll change it.

@ricardoV94 For an example with resizing `y` alone first, would `y` be a `pm.MutableData` object, and observed?  Makes sense RE fitting the same model to different datasets.  In the past I've just remade the model object but you could save on compiling time.

Reply:
Yes